src/BUILD

```diff
@@ -154,3 +154,13 @@ py_library(
     ],
     visibility = ["//visibility:public"]
 )
+
+py_test(
+    name = "test_receive_data",
```

Comment:
Is there a test suite that pulls together all individual tests like this one? If so, please hook this one in. If not, there should be. The CI should run that mega-suite.

Reply:
I'll open another PR for tests